docs/variables.md

```diff
@@ -113,6 +113,13 @@ Targeting service crn:v1:bluemix:public:power-iaas:tor01:a/65b64c1f1c2XXXXX:4a1f
 [question] > Enter the NFS volume size(GB) (300)
 ?
 - You have answered: 300
+[question] > Select the flavor for the cluster nodes:
+1) large.tfvars
```

Comment:
Why the flavor names have .tfvars in it? Please correct it in code as well.

Reply:
@yussufsh Updated the changes.